### PR TITLE
feat: handle operator payout address change in get_protx_listdiff

### PR DESCRIPTION
**Pull request by markin-io**
Include `operatorPayoutAddress` into `get_protfx_listdiff` response.

### CODE REVIEW

File: json/src/lib.rs [hunk 1]
```diff
@@ -2161,7 +2162,18 @@ impl TryFrom<DMNStateDiffIntermediate> for DMNStateDiff {
                 })
             })
             .transpose()?;
-        let operator_payout_address = None; //todo
+        let operator_payout_address = operator_payout_address
```
Comment (lklimek) [inline json/src/lib.rs:2165]: My first  impression is that processing of `operator_payout_address` is different than `payout_address` above (two  maps  here), copy-paste issue or expected behavior?

**Review by shumkov — APPROVED** [review]
👍

File: json/src/lib.rs (hunk 1 above)
Comment (lklimek) [reply json/src/lib.rs:2165]: My point is that:

* `payout_address` takes 9 lines of code, including one `map` operation
* `operator_payout_address` takes 12 lines of code, including two `map` operations

If they are the same type, I would expect both to take 9 lines of code and one `map` operation.

Comment (lklimek) [reply json/src/lib.rs:2165]: Basically, there should be a helper function that you will use for both cases (and other similar, like `voting_address`?).

**Pull request merged**